packages/cfd/src/Containers/__tests__/cfd-financial-stp-real-account-signup.spec.js

```diff
@@ -0,0 +1,191 @@
+            has_secret_answer: 1,
+            immutable_fields: ['residence'],
+            is_authenticated_payment_agent: 0,
+            last_name: 'am',
+            non_pep_declaration: 1,
+            phone: '+651213456',
+            place_of_birth: null,
+            preferred_language: 'EN',
+            request_professional_status: 0,
+            residence: 'Singapore',
+            salutation: '',
+            tax_identification_number: null,
+            tax_residence: null,
+            user_hash: '823341c18bfccb391b6bb5d77ab7e6a83991f82669c1ba4e5b01dbd2fd71c7fe',
+        },
+        is_fully_authenticated: true,
+        landing_company: {
+            config: {
+                tax_details_required: 1,
+                tin_format: ['^\\d{15}$'],
+                tin_format_description: '999999999999999',
+            },
+            dxtrade_financial_company: {},
+            dxtrade_gaming_company: {},
+            financial_company: {},
+            gaming_company: {},
+            id: 'id',
+            minimum_age: 18,
+            mt_financial_company: {},
+            mt_gaming_company: {},
+            name: 'Indonesia',
+            virtual_company: 'virtual',
+        },
+        residence_list: [
+            {
+                identity: {
+                    services: {
+                        idv: {
+                            documents_supported: {},
+                            has_visual_sample: 0,
+                            is_country_supported: 0,
+                        },
+                        onfido: {
+                            documents_supported: {
+                                passport: {
+                                    display_name: 'Passport',
+                                },
+                            },
+                            is_country_supported: 0,
+                        },
+                    },
+                },
+                phone_idd: '93',
+                text: 'Afghanistan',
+                value: 'af',
+            },
+        ],
+        states_list: {
+            text: 'Central Singapore',
+            value: '01',
+        },
+    };
+
+    it('should render CFDFinancialStpRealAccountSignup component', () => {
+        render(<CFDFinancialStpRealAccountSignup {...mock_props} />);
+
+        expect(screen.getByTestId('dt_cfd_financial_stp_modal_heading')).toBeInTheDocument();
+    });
+
+    it('should show headers properly', () => {
+        render(<CFDFinancialStpRealAccountSignup {...mock_props} />);
+
+        expect(screen.getByText('Complete your personal details')).toBeInTheDocument();
+        expect(screen.getByText('1')).toBeInTheDocument();
+        expect(screen.getByText('Personal details')).toBeInTheDocument();
+        expect(screen.getByText('Proof of identity')).toBeInTheDocument();
+        expect(screen.getByText('2')).toBeInTheDocument();
+        expect(screen.getByText('Proof of address')).toBeInTheDocument();
```

Comment:
I suggest putting 'Personal details', 'Proof of identity' & 'Proof of address' strings (and any other repeatedly used strings) into variables at the beginning of describe and use variables because these strings are used more than once each. It will help increase maintainability of these tests since it's gonna be easier to change them later if needed in case we don't hardcode strings inside test cases 🙂

Reply:
done